src/store/contracts/actions.ts

```diff
@@ -1,9 +1,12 @@
 import { createAsyncThunk, createAction } from '@reduxjs/toolkit';
 import { Contract } from '@models/contract.interface';
 import { fetchContract as fetchContractFromApi } from '@common/api/contracts';
+import { selectCurrentNetworkUrl } from '@store/ui/selectors';
 
-export const fetchContract = createAsyncThunk<Contract, string>('contracts/fetch', async query =>
-  fetchContractFromApi(query)
+export const fetchContract = createAsyncThunk<Contract, string>(
+  'contracts/fetch',
+  // @ts-ignore
```

Comment:
What's the alternative to using ts-ignore here? Perhaps this is a militant statement, but imo we should _never_ be using this.

Reply:
same issue https://github.com/blockstack/explorer/pull/134/files/878ab1d54796294bb3e53a0cd3923367e0b8345e#r426730594